apps/emqx_authz/include/emqx_authz.hrl

```diff
@@ -38,6 +17,9 @@
 -define(ACL_TABLE_USERNAME, 1).
 -define(ACL_TABLE_CLIENTID, 2).
 
+-type(action() :: subscribe | publish | all).
```

Comment:
Nit: it would be nice to move export these types from some module instead of keeping them in a hrl file